src/platform/pdf/templates/utils.js

```diff
@@ -365,7 +371,7 @@ const createDetailItem = async (doc, config, x, item) => {
           .fontSize(config.text.size)
           .text(titleText, x, doc.y, paragraphOptions);
         doc
-          .font(config.text.font)
+          .font(item.monospace ? config.text.monospaceFont : config.text.font)
```

Comment:
What will happen here if a monospace font is not specified in the template? Maybe it would be good to fall back on the main font if one isn't?

Reply:
Good point!